wireguard-server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:bullseye-slim
+
+RUN apt-get update && apt-get install -y wireguard iptables iproute2 curl systemd
+
+COPY wg-wirescale.conf /etc/wireguard/
+COPY entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
+
+# Install wgrest 
+COPY install-wgrest.sh /install-wgrest.sh
+RUN chmod +x /install-wgrest.sh
+RUN /install-wgrest.sh
+
+# Expose the WireGuard port
+EXPOSE 51820/udp
+
+# Set a simple static authentication token shared with the Wirescale API
+ENV WGREST_STATIC_AUTH_TOKEN=secret
```

Comment:
This means it will be visible in the env of the docker, which is anti-pattern, better use secret:

https://docs.docker.com/engine/swarm/secrets/

Reply:
For sure -- I was planning to use a file mount or similar when there was a need for sharing it between containers (i.e. when I add the code that uses it in the Wirescale API).

Definitely not pretty to have this in the env and I've pushed a change to use secrets instead. Thanks!